### PR TITLE
Feat/cal 95/date range on event types

### PR DESCRIPTION
**Pull request by femyeda**

### CODE REVIEW

File: components/ui/Text/Text.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    | "caption2"
+    | "footnote"
+    | "headline"
+    | "largetitle"
+    | "subheadline"
+    | "subtitle"
+    | "title"
+    | "title2"
+    | "title3";
+  children: any;
```
Comment (baileypumfleet) [inline components/ui/Text/Text.tsx:31]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734246)
```suggestion
  children: unknown;
```

**Comment by femyeda** [comment]
This will need a production migration. Not sure what's the process for that.

File: components/ui/Text/Text.tsx (hunk 1 above)
Comment (baileypumfleet) [inline components/ui/Text/Text.tsx:31]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734246)
```suggestion
  children: unknown;
```

Comment (baileypumfleet) [inline components/ui/Text/Text.tsx:31]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734246)
```suggestion
  children: unknown;
```

Comment (baileypumfleet) [inline components/ui/Text/Text.tsx:31]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734246)
```suggestion
  children: unknown;
```

**Pull request merged**